src/python/CMakeLists.txt

```diff
@@ -255,8 +255,11 @@ if(PYTHONINTERP_FOUND)
 
     # Specific for Mac
     if (${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
-        set(GUDHI_PYTHON_EXTRA_COMPILE_ARGS "${GUDHI_PYTHON_EXTRA_COMPILE_ARGS}'-mmacosx-version-min=10.14', ")
-        set(GUDHI_PYTHON_EXTRA_LINK_ARGS "${GUDHI_PYTHON_EXTRA_LINK_ARGS}'-mmacosx-version-min=10.14', ")
+      # if MACOSX_DEPLOYMENT_TARGET environment variable is set
```

Comment:
Isn't CMAKE_OSX_DEPLOYMENT_TARGET always defined? (not an issue, just surprised by the comment)

Reply:
I was not aware of this behavior, but yes you are right, it seems CMAKE_OSX_DEPLOYMENT_TARGET is always set on OSx. I reworked the comment on [ee221bd](https://github.com/GUDHI/gudhi-devel/pull/956/commits/ee221bd58fc5cb4b9eb575660d758f8651d00217)